src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+.stars__body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
```

Comment:
Consider using a variable for repeated values such as the width and height of `.stars_star`. This will make it easier to maintain and change the size of the stars in one place if needed.